Library/Formula/tinyproxy.rb

```diff
@@ -83,7 +83,7 @@ def plist; <<-EOS.undent
         <false/>
         <key>ProgramArguments</key>
         <array>
-            <string>#{opt_sbin}/tinyproxy</string>
+            <string>#{opt_sbin}/tinyproxy -d</string>
```

Comment:
`plists` tend to handle spacing a bit weirdly. Can you drop this onto another line instead?
```
<string>#{opt_sbin}/tinyproxy</string>
<string>-d</string>
```